portal-api/src/v1/validation/checker-submit-deal-facility-ready-for-check-cover-start-dates/index.js

```diff
@@ -3,21 +3,36 @@ const { orderNumber } = require('../../../utils/error-list-order-number');
 const { formattedTimestamp } = require('../../facility-dates/timestamp');
 const CONSTANTS = require('../../../constants');
 
-const facilityReadyForCheckRequestedCoverStartDateRule = (facility, errorList) => {
+const facilityReadyForCheckRequestedCoverStartDateRule = (deal, facility, errorList) => {
   const {
     requestedCoverStartDate,
     status,
   } = facility;
+  const { details, submissionType } = deal;
+  const { submissionDate, manualInclusionNoticeSubmissionDate } = details;
+  let dateOfSubmission;
```

Comment:
I think it will be clear to instantiate `dateOfSubmission` with `dateOfSubmission = moment().utc().valueOf();` and those else branches will only be reached when MIA.

What do you think?

Reply:
@IgnatG thoughts?

Reply:
done